app/assets/stylesheets/application.css

```diff
@@ -18,44 +18,42 @@
  @tailwind components;
  @tailwind utilities;
 
-
-
  .occupation-tabs *:focus, .occupation-tabs input:focus + label {
```

Comment:
We may not need this change as part of this PR

Note: it seems we don't have an standard regarding indentation. We should set something to handle that now that there is five people in the project

Reply:
I can remove the indentation change.